packages/release/Cargo.lock

```diff
@@ -0,0 +1,570 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I wonder if these are being generated by an editor plugin like rust-analyzer? We may want to concoct a .gitignore for these entries.

Reply:
Thats exactly what is happening